assessment/displayq2.php

```diff
@@ -2756,8 +2756,8 @@ function makeanswerbox($anstype, $qn, $la, $options,$multi,$colorbox='') {
 						if (count($answerformat)>1 && $answerformat[1]=='opendot') { $out .= 'class="sel" '; $def = 2;}
 						$out .= ' alt="Open dot"/>';
 					}
-					

```

Comment:
I'm a git noob, so I'm not sure why I'm generating line additions/deletions when committing... Only intended to change line 244.